tests/conftest.py

```diff
@@ -46,9 +46,12 @@ def myclient():
     # narrowly scoped to this function
     version_url = f"{TEST_LOCAL_SERVER}/segmentation/api/version"
     responses.add(responses.GET, version_url, json="2.15.0", status=200)
-
     client.chunkedgraph  # this will trigger the version check
 
+    mat_version_url = f"{TEST_LOCAL_SERVER}/materialize/version"
+    responses.add(responses.GET, mat_version_url, json="4.30.1", status=200)
+    client.materialize
```

Comment:
this is i think a mistake

Reply:
can you elaborate? i put this in bc now the client sees there's an endpoint for version and tries to check it